packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -12,7 +12,7 @@ type TMobileDialog = {
     has_content_scroll?: boolean;
     portal_element_id: string;
     renderTitle?: () => string;
-    title?: string;
+    title?: string | React.ReactNode;
```

Comment:
Add `React.ReactNode` in order to have an opportunity to pass title with tooltip (according to design)